db/seeds.d/50_discovery_templates.rb

```diff
@@ -1,8 +1,8 @@
-kind = TemplateKind.find_or_create_by_name('kexec')
+kind = TemplateKind.find_or_create_by(:name =>'kexec')
 
 ProvisioningTemplate.without_auditing do
   content = File.read(File.join(ForemanDiscovery::Engine.root, 'app', 'views', 'foreman_discovery', 'redhat_kexec.erb'))
-  tmpl = ProvisioningTemplate.find_or_create_by_name(
+  tmpl = ProvisioningTemplate.find_or_create_by(
```

Comment:
This is not exactly the same, the original one was finding *by name*, and creating it if it doesn't exist. This one is finding *by all of these attributes*, then creating it if doesn't exist.

`.where(:name => 'Discovery Red Hat kexec').first_or_create(:template_kind_id => kind.id, etc...)` would be a more exact equivalent